upgrade/upgrade-1.2.12.php

```diff
@@ -30,5 +32,42 @@
  */
 function upgrade_module_1_2_12($module)
 {
-    return $module->registerHook('displayInvoiceLegalFreeText');
+    foreach (PrestaShop\Module\PrestashopCheckout\OrderStates::ORDER_STATES as $state => $color) {
+        $orderStateId = (int) \Configuration::get($state);
+        setStateIcons($state, $orderStateId, $module->name);
+    }
+
+    return \Db::getInstance()->execute('ALTER TABLE `' . _DB_PREFIX_ . TableManager::TABLE_ORDER_MATRICE . '` CHANGE `id_order_prestashop` `id_order_prestashop` INT(10) UNSIGNED NOT NULL;')
```

Comment:
This come from upgrade file of previous version, it's necessary to run it another time ?

Reply:
ye true wasn't sure during rebase, fixed.